lib/improver/cli/nowcast_accumulate.py

```diff
@@ -98,19 +93,16 @@ def process(input_cube: cli.inputcube,
     from improver.nowcasting.forecasting import CreateExtrapolationForecast
     from improver.utilities.cube_manipulation import merge_cubes
 
-    u_cube = advection_cubes.extract(
-        Constraint("precipitation_advection_x_velocity"), True)
-    v_cube = advection_cubes.extract(
-        Constraint("precipitation_advection_y_velocity"), True)
+    u_cube, v_cube = advection_velocity
```

Comment:
How do you know these cubes will be in this order in the cubelist?

Reply:
When passed in through clize, they have the same order as was used in `create_constrained_inputcubelist_converter`.
(This is not necessarily true when calling directly with python though)

Reply:
Does that mean this change is ok or not?

Reply:
Because the order of returned elements follows the order of constrains used to create the `inputadvection` converter at the top of the file.